Update documentation to introduce `Subscriptions` (#150)
commercelayer/mfe-my-account: packages/my-account/README.md

- The payment method used for the order.
  
![my-account-order-details](https://github.com/commercelayer/mfe-my-account/assets/105653649/4d9869d0-7ae8-4e0a-ae72-a81b9f4aca7f)

#### Parcel tracking details

By clicking a parcel's tracking link you can enter the selected parcel's detail page showing the full timeline of the parcel's movements and updates.

![my-account-parcel-tracking](https://github.com/commercelayer/mfe-my-account/assets/105653649/fdab7860-5605-47ad-9011-3a66c8f23050)

### Addresses

#### Addresses Management

By clicking on the _Addresses_ menu item you can see the list of your saved addresses (if any). Each address can be edited or deleted. New addresses can be directly created from the app and added to the list.

![my-account-select-address](https://github.com/commercelayer/mfe-my-account/assets/105653649/7f32c95a-1292-4451-942d-8affdc909b02)

![my-account-edit-address](https://github.com/commercelayer/mfe-my-account/assets/105653649/9500665a-cc71-4012-af83-8082fdccbd8c)

### Subscriptions

#### Subscriptions history

By clicking on the _Subscriptions_ menu item you can see the history of your subscriptions (if any).

IMAGE

> The list is paginated and can be sorted based on each column piece of information:

Author: If we are ok with this change the same paragraph related to the orders needs to be aligned as well.

> The list is paginated and can be sorted based on each column piece of information:

Reviewer: let's see also what @sfiorucci thinks

> The list is paginated and can be sorted based on each column piece of information:

Reviewer 2: not a big deal, both are correct... your call.

- subscription number (default)
- subscription status
- subscription frequency
- subscription next run date

#### Subscription details

By clicking on one of the subscription numbers in the history list you can enter the selected subscription's details page where you can see:

- An introductory recap about subscription settings (like start date, frequency, next run date and address)
- The subscription summary information with the list of subscription line items.
- The payment method used for the subscription renewal.
- The history of subscription's related orders.
  
IMAGE

> It could happen that for any kind of reason the subscription is no longer able to be renewed, for example because of the expiry of the associated payment method.

Author: Ok! Applied!

> In this case, an alert is shown at the top of detail page to highlight the issue and provide the customer with a valid checkout link to place the order with a valid payment method and let the subscription payment method be updated for future runs.

Reviewer: ```suggestion
This alert highlights the issue and provides a link for the customer to checkout the order with a valid payment method, thereby updating the payment information for future transactions.
```

Author: Ok! Applied with a small change suggested by grammar check. I wrote `...provides a checkout link for the customer to place the order...`. Do you think is ok?
